utils/lavalog.go

```diff
@@ -71,20 +80,22 @@ func LavaFormatLog(description string, err error, attributes []Attribute, severi
 
 	var logEvent *zerolog.Event
 	switch severity {
-	case 4:
+	case LAVA_LOG_PANIC:
+		// prefix = "Panic:"
+		logEvent = zerologlog.Panic()
```

Comment:
does panic actually panics?

Reply:
> does panic actually panics?

time for some existential thoughts...

`zerologlog.Panic()` returns a logEvent, it doesn't panic.
the subsequent logging of the resulting logEvent implies panic() by the logging library.

Reply:
not sure I entirely understand. The logger implies panic() but it doesn't really happen?

Reply:
calling `utils.LavaFormatPanic(msg, err, ...)` will spit an error to the log and then `panic(msg)`.